packages/preview-middleware-client/src/adp/quick-actions/load.ts

```diff
@@ -8,6 +8,10 @@ import type { ApplicationType } from '../../utils/application';
  * @returns Quick Action registries.
  */
 export async function loadDefinitions(appType: ApplicationType): Promise<QuickActionDefinitionRegistry<string>[]> {
+    // eslint-disable-next-line fiori-custom/sap-no-localstorage
+    if (localStorage.getItem('com.sap.ux.control-property-editor.features.quick-actions') !== 'true') {
```

Comment:
One suggestion would be using cookie instead of local storage so we can avoid feature toggle for different port numbers.

Reply:
Cookies will also be sent to the server on every request, which we do not need in this case. Also having origin specific feature toggles seems to be more flexible - if you want to enable it only for a specific origin.